src/Containers/BikeMap/BikeMap.js

```diff
@@ -37,10 +39,10 @@ export class BikeMap extends Component {
   }
 
   render () {
-    // This setTimeout is needed in order for map to render tiles properly. Need css modules to load first.
-    setTimeout(() => {
-      this.setState({ loading: false })
-    }, 100);
+    // This setTimeout is needed in order for map to render tiles properly. Found this suggested fix online - something about needing css modules to load first. It slows down the map a ton though.
+    // setTimeout(() => {
```

Comment:
This setTimeout is needed in order for map to render tiles properly. Found this suggested fix online - something about needing css modules to load first. It slows down the map a ton though.

Reply:
I would put that in componentDidMount, not render - that way it only causes that delay once on the initial render.